resource/translations/skosmos_pt_BR.po

```diff
@@ -0,0 +1,770 @@
+# 
+# Translators:
+# Bruno P. Kinoshita <brunodepaulak@yahoo.com.br>, 2016,2020
+# Ronnie Fagundes de Brito <ronniefbrito@gmail.com>, 2018
+msgid ""
+msgstr ""
+"Project-Id-Version: Skosmos\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-01-16 10:26+0200\n"
+"PO-Revision-Date: 2020-05-05 10:44+0000\n"
+"Last-Translator: Bruno P. Kinoshita <brunodepaulak@yahoo.com.br>\n"
+"Language-Team: Portuguese (Brazil) (http://www.transifex.com/national-library-of-finland/skosmos/language/pt_BR/)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: pt_BR\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Poedit 1.8.7.1\n"
+"X-Poedit-Basepath: ../..\n"
+"X-Poedit-SearchPath-0: view\n"
+"X-Poedit-SearchPath-1: controller\n"
+"X-Poedit-SearchPath-2: model\n"
+"X-Poedit-SourceCharset: utf-8\n"
+
+msgid "Download this vocabulary:"
+msgstr "Baixar este vocabulário:"
+
+#: controller/Controller.php:38
+#: /tmp/cache/587c83c09abaa/4e/4e05e5051b20c875609479922a558d59b8500ae386eaf79bc460e69d8781a753.php:21
+msgid "in_this_language"
+msgstr "em Inglês"
```

Comment:
this should be the equivalent of "in Portuguese" or perhaps more likely "in Brazilian Portuguese" - this is what is shown as the label of the link to change the UI language

Reply:
Aaahhh